pages/k8s/charm-easyrsa.md

```diff
@@ -0,0 +1,131 @@
+---
+wrapper_template: "kubernetes/docs/base_docs.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "EasyRSA charm"
+  description: Kubernetes-master Charm reference
+keywords: kubernetes-master, charm, config
+tags: [reference]
+sidebar: k8smain-sidebar
+permalink: charm-easyrsa.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+
+
+This charm delivers the EasyRSA application to act as a Certificate Authority
+(CA) and creates certificates for related charms.
+
+EasyRSA is a command line utility to build and manage Public Key
+Infrastructure (PKI) Certificate Authority (CA).
+
+The purpose of a Public Key Infrastructure (PKI) is to facilitate the secure
+electronic transfer of information.
+
+## Deployment
+You can deploy an EasyRSA charm with Juju:
+
+```
+juju deploy easyrsa
+juju deploy tls-client
+juju add-relation easyrsa tls-client
+```
+
+## Using the easyrsa charm
+
+The easyrsa charm will become a Certificate Authority (CA) and generate a CA
```

Comment:
I think we should use EasyRSA everywhere except in code blocks where we're using the actual name of the charm.